1zadanie.css

```diff
@@ -0,0 +1,158 @@
+body{
+    padding: 0 auto;
+    margin: 0 auto;
+}
+
+#layout{
+    background: linear-gradient(90deg, #83EB94 0%, #47D95E 100%);
+    padding-bottom: 10px;
+    width: 100%;
+    padding-top: 40px;
+    align-items: center;
+}
+
+#coming_soon{
+    position: absolute;
+    right: 0%;
+    top:0%;
+  
+}
+
+#guy{
+    width: 70%;
+    position: relative;
+    left: 15vw;
+    filter: brightness(70%);
+    padding: 5px;
+}
+
+#code{
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: 300;
+    font-size: 60px;
+    text-align: center;
+    letter-spacing: 15px;
+    color: black;    
+    width: 100%;
+    
+}
+
+h3{
+    text-align: center;
+    color: #FFFFFF;
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: 200;
+    position: relative;
+    top:5px;
+    font-size: 24px;
+    width: 100%;
+}
+
+#get_mail{
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+    height: 80px;
+    margin-bottom: 200px;
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+     
+}
+
+#email_field{
+    
+    border-radius: 9px;
+    border-style: none;
+    box-shadow:inset 0 0 10px #000000;
+    font-family: 'Lato';
+    font-size: 16px;
+    text-indent: 10px;
+    font-style: normal;
+    font-weight: 300;
+    color: #A0A0A0;
+    font-weight: 300;
+    line-height: normal;
+    width: 250px;
+    height: 48px;
+   
+   
+}
+
+#button{
+    
+    background:#0297EC;
+    border-radius: 9px;
+    padding: 13px;
+    color:#FFFFFF;
+    border-style: none;
+    padding-left: 30px;
+    padding-right: 30px;
+    font-size: 16px;
+    text-align: center;
+    font-family: 'Lato';
+    width: 250px;
+    height: 48px;
+    margin: 0;
+    position: relative;
+    bottom: 10px;
+    
+    
+
+ 
+}
+
+#foot{
+    
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: 300;
+    text-align:center;
+    color: #FFFFFF;
+    width: 100%;
+    font-size: 24px;
+    line-height: 28px;
+}
+.sexiflexi{
```

Comment:
Za nic nie mogę się domyśleć co to jest... Pisz nazwy klas bardziej mówiące o tym, co jest czym ;)